components/prism-cpp.js

```diff
@@ -9,4 +9,12 @@ Prism.languages.insertBefore('cpp', 'keyword', {
 		pattern: /(class\s+)\w+/i,
 		lookbehind: true
 	}
+});
+
+Prism.languages.insertBefore('cpp', 'string', {
+	'raw-string': {
+		pattern: /R"([^()\ ]{0,16})\([\s\S]*\)\1"/,
```

Comment:
There is no need to escape the space char in the char class, is there?
Also, the part in the parentheses looks greedy: does this mean `R"(foo)"R"(bar)"` is a single string?

Reply:
Hi @Golmote, thanks for the review. Yes you are correct. My intention was not to escape the space char, but to exclude the backslash. It should be `\\` instead of a single `\`. You are also correct in that `R"(foo)"R"(bar)"` should not be a single string. I guess my regex skills are getting a bit rusty 😄 I'll fix it...